docs/components/zeebe/technical-concepts/health.md

```diff
@@ -0,0 +1,135 @@
+---
+id: health
+title: "Health"
+description: "This document explains how health is defined in a Zeebe cluster."
+---
+
+# Health
+
+Health in Zeebe is not a binary status, but can have three different state:
+
+- **Healthy**: everything is working as expected.
+- **Unhealthy**: at least parts of the system are in a degraded state.
+- **Dead**: one or more parts of the cluster experienced non-recoverable failure.
+
+## Unhealthy
+
+When a node in a Zeebe cluster is unhealthy, it means parts of the system may not work. This is often a transient state: for example, when a node is
+starting, it starts in an unhealthy state, as several components (e.g. a partition, the workflow engine, etc.) are not yet installed or ready for
+operation.
+
+When this is temporary, and the cluster overall converges to a steady healthy state, you can safely ignore these temporary signals. However, if things
+persist, or if you see flip-flopping between healthy and unhealthy, it can be an indicator of an underlying issue which requires investigation.
+
+:::note
+Note that even if parts of the cluster are unhealthy, it can be that other parts are still working fine.
+:::
+
+## Dead
+
+When something is marked as dead in a cluster, it means it failed in a non-recoverable way. This means it **will** require human intervention to recover.
+This is a rare status, but it can happen if data corruption is detected, for example. We recommend prompt investigation into any components with a dead
+status.
+
+:::note
+Note that it's possible that only parts of the system has failed, and the rest is still working fine. For example, A dead broker can be due to a single
+partition which has data corruption; other partitions on the same broker may still be processing and working as expected.
+:::
+
+## Cluster
+
+The health of a cluster depends on the deployment topology. More specifically, the health of a cluster is based on the expected number of brokers,
+gateways, partitions, and the replication factor.
+
+:::note
+If using the single application, or embedded gateways - that is, where brokers act as gateways - then you can ignore the gateway part.
+:::
+
+:::note
+Starting with 8.8, we consider a `gateway` to be anything which exposes the Camunda 8 REST API, and optionally, the Camunda 8 gRPC API. While this is
+often the Zeebe Gateway, it can be a single application combining Operate, Tasklist, and the Zeebe Gateway.
+:::
+
+Informally, a healthy cluster is one where the the expected number of
+[brokers and gateways report a healthy status](../../../self-managed/zeebe-deployment/operations/health.md), where every partition has exactly one leader, and
+there are `N-1` followers per partition (where `N` is the replication factor).
+
+More formally, given U the number of expected gateways, X the number of expected brokers, Y the partition count, and Z the replication factor, we can
```

Comment:
:thought_balloon:  hard to keep track of what U,X,Y,Z are:
```
U -> G
X -> B
Y -> P
Z -> R
```
may be more intuitive :shrug:

Reply:
I had a really hard time not using "math class" language, but describing it more naturally... :smile: